src/Transports/MassTransit.AmazonSqsTransport/Configuration/AmazonSqsBusFactoryConfiguratorExtensions.cs

```diff
@@ -38,5 +39,18 @@ public static void LocalstackHost(this IAmazonSqsBusFactoryConfigurator configur
                 h.Config(new AmazonSimpleNotificationServiceConfig { ServiceURL = "http://localhost:4566" });
             });
         }
+
+        public static void UseDefaultHost(this IAmazonSqsBusFactoryConfigurator configurator)
+        {
+           configurator.UseDefaultHost(FallbackRegionFactory.GetRegionEndpoint());
```

Comment:
Any reason not to make this the default in the `AmazonSqsRegistrationBusFactory`, similar to how RabbitMQ has a default host configuration to localhost?

Reply:
Only reason is I didn't look harder at the Rabbit host configuration, I'll look harder and update